rflx/model.py

```diff
@@ -626,67 +632,49 @@ def __target_length(self, link: Link) -> Expr:
     def __target_last(self, link: Link) -> Expr:
         return Sub(Add(self.__target_first(link), self.__target_length(link)), Number(1))
 
-    def __link_expression(self, link: Link) -> Expr:
+    @classmethod
+    def __expression_list(cls, expr: Expr) -> Sequence[Expr]:
+        # splitting conjunctions helps getting better error messages
+        if isinstance(expr, And):
+            return expr.terms
+        return [expr]
```

Comment:
That shouldn't be a class method. Either use a static method or a separate function. I tend to the latter.

Reply:
Fixed in 5522181.